serpent/serpent_input.i

```diff
@@ -0,0 +1,456 @@
+Simplified model of a reactor
+% Structure level
+%   Pin level
+%     - master pin
+%     - copy pins
+%   Bundle level
+%     - Fuel bundles
+%     - Bottom adapters
+%     - Reflector elements
+%   Core level
+%     - Transient rod
+%     - Control shrouds
+%     - Safety blades
+%     - Regulating blade
+%     - Core lattices
+%     - Core regions
+%   Reactor level
+%     - Reactor pool
+%     - Thermal column
+%     - Beam ports
+%     - Concrete shield
+% 
+%  -------------------------------  CELL CARD  ------------------------------  C
+%  Pin level
+%    Master fuel pin
+%    Row:3 Column:1 Pin:2
+cell 1310200  3102  1190100  1000 -1001 1100 -1101                   %Fuel_R1_A1
+cell 1310201  3102  1190101  1000 -1001 1101 -1102                   %Fuel_R1_A2
+cell 1310202  3102  1190102  1000 -1001 1102 -1103                   %Fuel_R1_A3
+%      Active cladding region
+cell 2310200  3102  1190199  1001 -1002 1100 -1103                      %Clad_A1
+%      Non-active region
+cell 2310292  3102  50002  -1002 2002 -2003                     %Top_end_fitting
+cell 2310293  3102  50002  1001 -1002 1103 -2002                  %Top_refl_clad
+cell 2310294  3102  30001  -1001 1103 -2002                       %Top_refl_meat
+cell 2310295  3102  20000  -1000 1100 -1103                              %Zr_rod
+cell 2310296  3102  30001  -1001 2001 -1100                       %Bot_refl_meat
+cell 2310297  3102  50002  1001 -1002 2001 -1100                  %Bot_refl_clad
+cell 2310298  3102  50002  -1002 2000 -2001                     %Bot_end_fitting
+cell 2310299  3102  30000  (1002:-2000:2003)                     %External_water
+trans 3102 2.5 2.5 0
```

Comment:
This **trans** cards are to move the fuel pin to the center of the geometry(0.0, 0.0) because as the surface defines the fuel at (-2.5, -2.5). The Serpent lattice requires Serpent fuel pins all defined as the original point.